.pre-commit-config.yaml

```diff
@@ -0,0 +1,64 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-json
+      - id: forbid-submodules
+      - id: double-quote-string-fixer
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: destroyed-symlinks
+      - id: detect-private-key
+      - id: pretty-format-json
+        args: ['--no-sort-keys', '--autofix', '--indent=2']
+      - id: mixed-line-ending
+        args: [ "--fix=lf" ]
+        description: Forces to replace line ending by the UNIX 'lf' character.
+      - id: no-commit-to-branch
+        args: [ --branch, master, --branch, main ]
+      - id: check-added-large-files
+        args: [ '--maxkb=500' ]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: "v8.29.0"
+    hooks:
+      - id: eslint
+        name: eslint
+        entry: eslint
+        language: node
+        types: [ javascript ]
+        types_or: [javascript, jsx, ts, tsx, vue]
+        args: [ --fix ]
+        require_serial: false
+        additional_dependencies:
+          - "@babel/core@7.15.8"
+          - "@babel/plugin-proposal-class-properties@7.10.4"
+          - "@babel/plugin-proposal-object-rest-spread@7.11.0"
+          - "@babel/plugin-transform-runtime@7.11.0"
+          - "@babel/polyfill@7.12.1"
+          - "@babel/preset-env@7.15.8"
+          - "babel-loader@8.1.0"
+          - "babel-plugin-add-module-exports@1.0.0"
+          - "babel-plugin-istanbul@4.1.1"
+          - "babel-plugin-syntax-jsx@6.18.0"
+          - "babel-plugin-transform-imports@1.4.1"
+          - "babel-preset-env@1.3.2"
+          - "babel-preset-stage-2@6.22.0"
+          - "@babel/eslint-parser@7.11.3"
+          - "eslint@7.6.0"
+          - "eslint-config-standard@14.1.1"
+          - "eslint-friendly-formatter@3.0.0"
+          - "eslint-loader@4.0.2"
+          - "eslint-plugin-import@2.22.0"
+          - "eslint-plugin-node@11.1.0"
+          - "eslint-plugin-promise@4.2.1"
+          - "eslint-plugin-standard@4.0.1"
+          - "eslint-plugin-vue@7.0.0-beta.2"
+          - "eslint-plugin-cypress@2.12.1"
```

Comment:
are all of these needed?
Is the `@babel/eslint-parser` using all the babel things?

I think some of those are redundant anyway.  But probably we tackle them in the dependency update issue

Reply:
Yup. Very, very hard to distinguish what's needed and what's not rn. I added all the deps bc the eslint parser was requesting them. It was an iterative process.